code/game/machinery/autolathe.dm

```diff
@@ -1,5 +1,5 @@
 // makes sure that discounted prices from upgraded lathe no less than 1 unit.
-#define SANITIZE_LATHE_COST(n) max(1, n) // helps to fix prices where "* mat_efficiency" is used.
+
```

Comment:
1) Why it was removed? 
2) Isn't it worth to add `round(..)` call here instead of copy-pasting values all over that file?

Reply:
I mean yeah values can now be less than 1, but that's not used as restriction from negative values too?

Reply:
I don't see any way negative values could possibly happen. but i could put the round in there, sure

Reply:
mat_efficency count is thin place. There was situation ... in short if someone add more part to autolathe it mat_efficiency can become negative. It's count that way:
`mat_efficiency = 1.1 - man_rating * 0.1`

But you right. That's not that important point, if values go below 0 we'll notice it immediately.